QuadradiusR/scripts/embedded_server.gd

```diff
@@ -39,7 +39,8 @@ func _process(delta):
 
 func run_server():
 	if not self.available:
-		push_error('Server executable is not available')
+#		push_error('Server executable is not available')
```

Comment:
- [ ] this should probably be uncommented before merging, i removed it for testing